### ISSUE TITLE
Add second node to cluster failed on AWS EC2

### ISSUE THREAD

**Issue by jin-tron** [ipfs/ipfs-cluster#826]
**Additional information:**
 - OS: [e.g. Linux] x86_64 GNU/Linux, AWS Linux AMI 2
 - IPFS Cluster version: [e.g. 0.10.1, master] master
 - Installation method: [e.g. built from source, dist.ipfs.io, docker] built from source

**Describe the bug:**
I have 2 AWS EC2 instance in same vpc and subnet. I followed doc/quickstart, first node installed successfully and cluster service is running fine.
When I add second node to cluster using ipfs-cluster-service daemon --bootstrap, I got error
ERROR    cluster: ***** ipfs-cluster consensus start timed out (tips below) ***** cluster.go:541
I have configured in both node service.json
"cluster": {
"listen_multiaddress": "/ip4/0.0.0.0/tcp/9096",

"ipfsproxy": {
      "listen_multiaddress": "/ip4/0.0.0.0/tcp/9095",
      "node_multiaddress": "/ip4/0.0.0.0/tcp/5001",

Detail log is as below, I enabled debug logs, looks to me error happened when do raft election,

DEBUG    cluster: bootstrap count 0 cluster.go:157
INFO    raftlib: Node at 12D3KooWHGRKXLAQcHdJQVRBYsKmHTfTnem3oi7yq9zQf5KpqmK1 [Follower] entering Follower state (Leader: "") logging.go:64
DEBUG       raft: checking for existing raft states raft.go:173
DEBUG       raft: staging servers do not need initialization raft.go:204
INFO       raft: peer is ready to join a cluster raft.go:205
INFO        dht: starting bootstrap query (0/1) to random ID (routing table size was 0) dht_bootstrap.go:151
INFO        dht: finished bootstrap query (0/1) to random ID (routing table size is now 0) dht_bootstrap.go:154
WARNI        dht: error bootstrapping: failed to find any peer in table dht_bootstrap.go:86
INFO    restapi: REST API (HTTP): /ip4/0.0.0.0/tcp/9094 restapi.go:457
DEBUG   addrutil: InterfaceAddresses: from manet: [/ip4/127.0.0.1 /ip4/172.31.21.65 /ip6/::1 /ip6/fe80::56:1cff:fe72:f0c0] addr.go:121
DEBUG   addrutil: InterfaceAddresses: usable: [/ip4/127.0.0.1 /ip4/172.31.21.65 /ip6/::1] addr.go:133
DEBUG   addrutil: adding resolved addr: /ip4/0.0.0.0/tcp/9096 /ip4/127.0.0.1/tcp/9096 [/ip4/127.0.0.1/tcp/9096] addr.go:64
DEBUG   addrutil: adding resolved addr: /ip4/0.0.0.0/tcp/9096 /ip4/172.31.21.65/tcp/9096 [/ip4/127.0.0.1/tcp/9096 /ip4/172.31.21.65/tcp/9096] addr.go:64
DEBUG   addrutil: ResolveUnspecifiedAddresses: [/p2p-circuit /ip4/0.0.0.0/tcp/9096] [/ip4/127.0.0.1 /ip4/172.31.21.65 /ip6/::1] [/p2p-circuit /ip4/127.0.0.1/tcp/9096 /ip4/172.31.21.65/tcp/9096] addr.go:109
INFO    restapi: REST API (libp2p-http): ENABLED. Listening on:
      /ip4/127.0.0.1/tcp/9096/ipfs/12D3KooWHGRKXLAQcHdJQVRBYsKmHTfTnem3oi7yq9zQf5KpqmK1
      /ip4/172.31.21.65/tcp/9096/ipfs/12D3KooWHGRKXLAQcHdJQVRBYsKmHTfTnem3oi7yq9zQf5KpqmK1
restapi.go:474
INFO  ipfsproxy: IPFS Proxy: /ip4/0.0.0.0/tcp/9095 -> /ip4/0.0.0.0/tcp/5001 ipfsproxy.go:278
INFO    service: Bootstrapping to /ip4/172.31.16.73/tcp/9096/ipfs/12D3KooWHGRKXLAQcHdJQVRBYsKmHTfTnem3oi7yq9zQf5KpqmK1 daemon.go:196
DEBUG    cluster: Join(/ip4/172.31.16.73/tcp/9096/ipfs/12D3KooWHGRKXLAQcHdJQVRBYsKmHTfTnem3oi7yq9zQf5KpqmK1) cluster.go:831
WARNI    raftlib: no known peers, aborting election logging.go:68
ERROR    cluster: ***** ipfs-cluster consensus start timed out (tips below) ***** cluster.go:541

I tried on two physical nodes in my local LAN, there's no issue to set up cluster and add second node.
I also tried to add address "/ip4/172.31.21.65/tcp/9096/ipfs/**" into .ipfs-cluster/peerstore, but not work either.

Do we have a document for AWS ipfs-cluster set up? I am guessing maybe I need configure the public IP of AWS node to somewhere to make ipfs-cluster work.
Thank you!

**Comment by jin-tron** [comment]
Forgot to add:
All ports are open in my AWS security group.

**Comment by jin-tron** [comment]
Sorry to report this issue, I think I found the issue in my AWS setup.
Because the ipfs-cluster build was broken in recent days, I made an image using the first node then boot second EC2 instance using that image.
The node id in ipfs-cluster is not updated when I run "ipfs-cluster-service init" because there's already a 
 identity.json comes from node1 image.

$ cat identity.json 
{
    "id": "12D3KooWHGRKXLAQcHdJQVRBYsKmHTfTnem3oi7yq9zQf5KpqmK1",

So these two nodes have same id in ipfs-cluster that made error.
After I deleted the .ipfs-cluster directory and  run "ipfs-cluster-service init", then 
ipfs-cluster-service daemon --bootstrap /ip4/172.31.16.73/tcp/9096/ipfs/12D3KooWHGRKXLAQcHdJQVRBYsKmHTfTnem3oi7yq9zQf5KpqmK1

Now my second node added to cluster and working fine.
